apps/iam/kinds/v0alpha1/rolespec.cue

```diff
@@ -15,6 +15,9 @@ RoleSpec: {
 	group: string
 	permissions: [...#Permission]
 
+	// Required for authz checks
+	internalID: int64
```

Comment:
From @stephaniehingtgen, we can probably use labels for a while, until we can use UIDs for access control.